node/pkg/driver/messages.go

```diff
@@ -18,3 +18,5 @@ package driver
 var ErrorWhileTryingToReadIQN = "Error while trying to get iqn  from string: %v."
 
 var ErrorUnsupportedConnectivityType = "Unsupported connectivity type : {%v}"
+
+var ErrorWhileTryingToReadFC = "Error while tring to get FC port from string: %v."
```

Comment:
from string?
please be more descriptive, since its a text that the client may see as error.